com.github.juliagoda.antimicro.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id": "com.github.juliagoda.antimicro",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "antimicro",
+    "finish-args": [
+        "--device=all",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--filesystem=home:rw",
+        "--socket=system-bus"
+    ],
+    "modules": [
+        {
+            "name": "antimicro",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DAPPDATA=ON",
+                "-DUSE_SDL_2=ON",
+                "-DWITH_UINPUT=OFF",
+                "-DWITH_XTEST=ON",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/juliagoda/antimicro.git"
```

Comment:
Having a specific commit or a release tag is better. Flathub is all about stable & reproducible builds.